### PR TITLE
JVM Toolchains dogfooding - part 1

### PR DESCRIPTION
**Pull request by ljacomet**
Leverage JVM toolchains in the Gradle build itself.

This first step runs tests with toolchains.

### CODE REVIEW

**Comment by ljacomet** [comment]
@bot-gradle test RFM

**Comment by ljacomet** [comment]
@bot-gradle test RFM

**Comment by ljacomet** [comment]
@bot-gradle test RFM

**Comment by ljacomet** [comment]
@bot-gradle test RFM

**Comment by ljacomet** [comment]
@bot-gradle test RFM

**Comment by ljacomet** [comment]
@bot-gradle test RFM

File: .teamcity/Gradle_Check/configurations/PerformanceTestsPass.kt [hunk 1]
```diff
@@ -18,10 +18,7 @@ package Gradle_Check.configurations
 
 import common.Os
 import common.applyDefaultSettings
-import configurations.BaseGradleBuildType
-import configurations.gradleRunnerStep
-import configurations.publishBuildStatusToGithub
-import configurations.snapshotDependencies
+import configurations.*
```
Comment (blindpirate) [inline .teamcity/Gradle_Check/configurations/PerformanceTestsPass.kt:21]: Nit: shouldn't use star import

**Review by blindpirate — APPROVED** [review]
LGTM! Only one tiny issue.

**Comment by ljacomet** [comment]
@bot-gradle test RFM

**Comment by ljacomet** [comment]
Obsolete, see #15926

**Pull request closed without merging**